### PR TITLE
Publish to PyPi and remove travis.yml.

### PR DESCRIPTION
**Pull request by AurelienJaquier**
Stop publishing to TestPyPi and use github actions to publish to PyPi.
Remove travis.yml to avoid publishing twice to PyPi.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-after_success:
-    - codecov -F unit -f cov_reports/unit.coverage.xml
-    - codecov -F functional -f cov_reports/functional.coverage.xml
-cache:
-  pip: true
-  directories:
-  - $TRAVIS_BUILD_DIR/.tox
-git:
-  depth: 2147483647
-deploy:
```
Comment (wvangeit) [inline .travis.yml:21]: Could you just keep the Travis tests for now ? I'll disable the requirement for these tests to pass.
But the deployment should be disabled.

Comment (AurelienJaquier) [reply .travis.yml:21]: Oh, ok. My bad.

**Comment by AurelienJaquier** [comment]
I also added a schedule to run tests every sunday at 3am.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
-name: Test and Publish sdist tarball to TestPyPi
+name: Test and Publish sdist tarball to PyPi
 
 on:
+  schedule:
+    # Runs every sunday at 3 a.m.
+    - cron: '0 3 * * SUN'
```
Comment (wvangeit) [inline .github/workflows/main.yml:6]: Do we have to specify Sunday? Just once a week would be good.

Comment (AurelienJaquier) [reply .github/workflows/main.yml:6]: We can specify a day of the week (MON-SUN) or a day of the month (1-31) or no day at all (-> runs each day). Setting it to Sunday seemed like a decent way to make it run once a week.

Comment (wvangeit) [reply .github/workflows/main.yml:6]: Ok

**Pull request merged**